ts/Series/Networkgraph/NetworkgraphSeriesDefaults.ts

```diff
@@ -216,6 +216,11 @@ const NetworkgraphSeriesDefaults: NetworkgraphSeriesOptions = {
         },
         style: {
             transition: 'opacity 2000ms'
+        },
+
+        defer: true,
+        animation: {
+            defer: 2000
```

Comment:
I am thinking 2 seconds is a bit much? On the [main demo](http://localhost:3030/samples/#view/highcharts/demo/network-graph), I would say 1000 works well.